workflow/rules/cnvkit_plot.smk

```diff
@@ -0,0 +1,40 @@
+# vim: syntax=python tabstop=4 expandtab
+# coding: utf-8
+
+__author__ = "Jonas Almlöf"
+__copyright__ = "Copyright 2021, Jonas Almlöf"
+__email__ = "jonas.almlof@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule cnvkit_plot:
```

Comment:
Cnvkit has several plotting options. Maybe it would be better to rename to `cnvkit_scatter` or allowing a parameter to produce the desired plot (scatter, heatmap or diagram)?

Reply:
Renamed the rule